### PR TITLE
Added middleware to handle auth errors

### PR DESCRIPTION
**Pull request by mwangimuringi**
export const errorHandler = (statusCode, message) => {
  const error = new Error(message);
  error.statusCode = statusCode;
  error.message = message;
  return error;
};

### CODE REVIEW

File: api/controllers/auth.controller.js
Comment (muringitech) [inline api/controllers/auth.controller.js:1]: ok

File: api/utils/error.js
Comment (muringitech) [inline api/utils/error.js:1]: Error to handle sign up error

File: api/index.js
Comment (muringitech) [inline api/index.js:1]: ok

**Review by muringitech — APPROVED** [review]
awesome

File: api/controllers/auth.controller.js
Comment (mwangimuringi) [reply api/controllers/auth.controller.js:1]: nice to know that

File: api/utils/error.js
Comment (mwangimuringi) [reply api/utils/error.js:1]: ok

File: api/index.js
Comment (mwangimuringi) [reply api/index.js:1]: nice

**Pull request merged**